### ISSUE TITLE
Bitget `fetchOpenOrders` requires symbol

### ISSUE THREAD

**Issue by faberkeping** [ccxt/ccxt#17880]
### Operating System

Windows 10

### Programming Languages

python

### CCXT Version

3.0.90

### Description

In the Bitget docs, it's shown that it has endpoints to fetch all open orders that do not requires  any `symbol`, for both spot and futures:
- [SPOT all orders](https://bitgetlimited.github.io/apidoc/en/spot/#get-order-list) **POST** endpoint: _/api/spot/v1/trade/open-orders_
- [FUTURES all orders](https://bitgetlimited.github.io/apidoc/en/mix/#get-all-open-order) **GET** endpoint: _/api/mix/v1/order/marginCoinCurrent_

> Each endpoint has 20 times/1s limit, which is the same API limit for the endpoint to fetch open orders using `symbol`

Is there any specific reason why the `fetch_open_orders()` function in CCXT for Bitget exchange requires `symbol` argument?

### Code

Try to fetch all open orders on bitget exchange without `symbol` argument
```
open_orders = exchange.fetch_open_orders()
```
The `ArgumentsRequired` exception was caught:
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "D:\Projects\kepingai\env\kaiborg-bitget\lib\site-packages\ccxt\bitget.py", line 2795, in fetch_open_orders
    self.check_required_symbol('fetchOpenOrders', symbol)
  File "D:\Projects\kepingai\env\kaiborg-bitget\lib\site-packages\ccxt\base\exchange.py", line 3746, in check_required_symbol
    self.checkRequiredArgument(methodName, symbol, 'symbol')
  File "D:\Projects\kepingai\env\kaiborg-bitget\lib\site-packages\ccxt\base\exchange.py", line 3726, in check_required_argument
    raise ArgumentsRequired(message)
ccxt.base.errors.ArgumentsRequired: bitget fetchOpenOrders() requires a symbol argument
```

**Comment by carlosmiei** [comment]
Hello @faberkeping, 
Yeah I think you're right. Thanks for letting us know; we will update it accordingly.

**Comment by faberkeping** [comment]
Hi @carlosmiei , I saw the [PR has been merged](https://github.com/ccxt/ccxt/pull/17881), does it mean the update is already available in the CCXT `v3.0.99`?

**Comment by faberkeping** [comment]
hi @kroitor , the latest CCXT version (`v3.0.100`) [still requires symbol for Bitget](https://github.com/ccxt/ccxt/blob/d9144b0884bef66a54944250011f0338e08bd479/python/ccxt/bitget.py#L2798) `fetch_open_orders()`, so I assume it should be available on the next version, `v3.0.101`?
